source/ExpandablePicker.js

```diff
@@ -370,6 +370,9 @@
 			this.inherited(arguments);
 			this.$.currentValue.setShowing(!this.open);
 			this.setActive(this.getOpen());
+			if(this.open) {
+				this.$.drawer.render();	
```

Comment:
@dmikey, did you try explicitly reflowing, instead of re-rendering? If we can get away with just a reflow, that would be preferable for performance reasons.